docs/get-started/developers/tokens.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: layout-with-subnav.njk
+title: Tokens
+heading: Developers
+tags:
+  - developers
+permalink: /get-started/developers/tokens/index.html
+order: 30
+bodyClasses: element-docs
+---
+
+## How to install tokens
+
+Run the following git command to install RHDS tokens:
+
+<rh-code-block>
+npm i @rhds/tokens
+</rh-code-block>
+
+## Usage
+
+We use [style-dictionary](https://amzn.github.io/style-dictionary/) to transform our tokens into multiple formats and helpers.
+
+### Import global CSS
+
+Apply defaults to the document root by importing the global stylesheet:
+
+<rh-code-block>
+    <script type="text/html">
+<link rel="stylesheet" href="/url/to/@rhds/tokens/css/global.css">
+<style>
+  :is(h1, h2, h3, h4, h5, h6) {
+    font-family: var(--rh-font-family-heading);
+  }
+</style>
+  </script>
+</rh-code-block>
+
+### Reset the shadowroot
+
+Reset a component's styles (preventing inheritance) by adding resetStyles to its static Constructible Style Sheet list:
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+import { resetStyles } from '@rhds/tokens/css/reset.css.js';
+import style from './rh-jazz-hands.css';
+
+@customElement('rh-jazz-hands')
+export class RhJazzHands extends LitElement {
+  static readonly styles = [resetStyles, style];
+}
+  </script>
+</rh-code-block>
+
+### Import tokens as JavaScript objects
+
+{% alert title="Note", state="info" %}
+We strongly recommend using CSS variables (and accompanying snippets), instead of importing tokens as JavaScript objects.
+{% endalert %}
+
+Import tokens as JavaScript objects:
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+import { tokens } from '@rhds/tokens';
+
+html`<span style="color: ${tokens.get('--rh-color-blue-300')}">I'm blue</span>`;
+  </script>
+</rh-code-block>
+
+or for tree-shakable imports:
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+import { ColorBlue300 } from '@rhds/tokens/values.js';
+
+html`<span style="color: ${ColorBlue300}">I'm blue</span>`;
+  </script>
+</rh-code-block>
+
+## Plugins
+
+### Using editor snippets
+
+Editor snippets complete prefixes like `--rh-color-brand` to their CSS custom properties, complete with fallback.
+
+<rh-code-block>
+  <script type="text/css">
+color: var(--rh-color-brand, #ee0000);
+  </script>
+</rh-code-block>
+
+They also provide reverse lookup. For example,  if you want to choose between all the tokens with the  value `#e00`, you can do so by completing the prefix `e00`.
+
+#### Load snippets in VSCode 
+
+Download the VSIX bundle that’s linked at the bottom of our [“Release v1.0.0”](https://github.com/RedHat-UX/red-hat-design-tokens/releases/tag/v1.0.0) page.
```

Comment:
Do we need a new VSIX bundle for tokens 2.0? Can we add the download link for the bundle from the README?

Reply:
That file doesn't link to the tag assets automatically IIRC.  I generated locally and uploaded for v2.0.1

See assets: https://github.com/RedHat-UX/red-hat-design-tokens/releases/tag/v2.0.1

Direct link to file:
https://github.com/RedHat-UX/red-hat-design-tokens/releases/download/v2.0.1/red-hat-design-tokens-2.0.1.vsix